Cargo.toml

```diff
@@ -19,6 +19,7 @@ doctest = false
 
 [dependencies]
 unicode-xid = "0.1"
+memchr = "2.0"
```

Comment:
With @Manishearth's work on improving the performance of the find method in https://github.com/rust-lang/rust/pull/46735, it probably won't be useful to have this dependency on memchr anymore, so we might want to drop it.

Reply:
Oh _that's_ why you wanted this.

Sneaky, sneaky 😺